testing/fuchsia/run_tests.sh

```diff
@@ -16,10 +16,11 @@
 
 set -Ee
 
+test_timeout_seconds=300
 
-echo "These tests have been temporarily disabled."
-echo "See: https://github.com/flutter/flutter/issues/57061"
-exit 0
+# This is longer than the test timeout as dumping the
+# logs can sometimes take longer.
+ssh_timeout_seconds=360
```

Comment:
Seems like in some cases the timeouts are happening after 2 mins. That may be a default timeout inside fuchsia_ctl we may need to take a look.

Reply:
Seems like you are already covering it with the logs collection timeout.

Reply:
On glancing at the Fuchsia CTL code, the timeout passed here as an arg should override any default timeouts set. I kicked off a couple of runs to make sure that this does fix the issue. I'll keep an eye out for any potential failures.